src/core/publizon/mutator/PublizonServiceHttpError.ts

```diff
@@ -0,0 +1,9 @@
+import FetcherCriticalHttpError from "../../fetchers/FetcherCriticalHttpError";
+
+// This HttpError is critical because we want to inform the user that something went wrong
+// while ordering a digitial article.
+export default class PublizonServiceHttpError<
+  ContextType
```

Comment:
One of us is confused here :smile:. As far as I know Publizon is not related to ordering of digital articles - at the moment at least.

Publizon has to do with checking the availability of digital materials and as such I do not see it as critical.